README.md

```diff
@@ -109,12 +109,44 @@ The template for the `blurb add` message looks like this:
 Here's how you interact with the file:
 
 * Add the GitHub issue number for this commit to the
-  end of the `.. gh-issue:` line.
+  end of the `.. gh-issue:` line. The issue can also
+  be specified via the ``-i/--issue`` option:
+
+  ```shell
+  $ blurb add -i 109198
+  # or equivalently
+  $ blurb add -i https://github.com/python/cpython/issues/109198
+  ```
 
 * Uncomment the line with the relevant `Misc/NEWS` section for this entry.
   For example, if this should go in the `Library` section, uncomment
   the line reading `#.. section: Library`.  To uncomment, just delete
-  the `#` at the front of the line.
+  the `#` at the front of the line.  Alternatively, the section can
+  be specified via the ``-s/--section`` option:
+
+  ```shell
+  $ blurb add -s 'Library'
+  # or equivalently
+  $ blurb add -s lib
+  ```
+
+  The known section names are given in the following table. The match
+  is performed casse insensitively and partial matching is supported
+  as long as the match is unique:
+
+  | Section Name      |
+  |-------------------|
+  | Security          |
+  | Core and Builtins |
+  | Library           |
+  | Documentation     |
+  | Tests             |
+  | Build             |
+  | Windows           |
+  | macOS             |
+  | IDLE              |
+  | Tools/Demos       |
+  | C API             |
```

Comment:
I think we can skip the table now there's no number mapping? We have the section names listed above in the template. Alternatively, this could just be a list.
```suggestion
  The match is performed case insensitively and partial matching is 
  supported as long as the match is unique.
```

Reply:
Mmmh.. you're totally right!